### PR TITLE
Optimizely data platform changes

### PR DESCRIPTION
**Pull request by joe-ayoub-segment**
Changes to ODP Destination

## Testing

Tested locally.

### CODE REVIEW

File: packages/destination-actions/src/destinations/optimizely-data-platform/emailEvent/index.ts [hunk 1]
```diff
@@ -27,6 +24,14 @@ const action: ActionDefinition<Settings, Payload> = {
         '@path': '$.properties.campaign_name'
       }
     },
+    campaign_id: {
```
Comment (Romani-Ramzi) [inline packages/destination-actions/src/destinations/optimizely-data-platform/emailEvent/index.ts:27]: We didnt add campaign_id to the body when sending.

Comment (joe-ayoub-segment) [reply packages/destination-actions/src/destinations/optimizely-data-platform/emailEvent/index.ts:27]: Hi @Romani-Ramzi - looks like it's not added into the payload being sent to ODP. 
I'll update this now. 

Is there anything else? Or are you happy for me to deploy this code?

Comment (Romani-Ramzi) [reply packages/destination-actions/src/destinations/optimizely-data-platform/emailEvent/index.ts:27]: Yes please, deploy

**Comment by joe-ayoub-segment** [comment]
This PR has been deployed.

**Pull request merged**